### PR TITLE
docs(Internal Systems):add documentation

### PR DESCRIPTION
**Pull request by Fletch99**
# Description
Added documentation for the Internal Systems team, including all research including images and available meeting minutes documents

## Type of change

- [ ] Documentation (new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from any of ginge270, tancnle, talzid and j-6 on the Pull Request

### CODE REVIEW

File: docs/Internal Systems/Research & Findings/JekyllResearchTushar.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Jekyll Research 
+
+_Jekyll is an open-source static site generator which is written in Ruby._
```
Comment (ginge270) [inline docs/Internal Systems/Research & Findings/JekyllResearchTushar.md:3]: I would probably change the file name to not include the author name as this documentation is for future teams, however you can detail the author in the file somewhere if you like.

Comment (ginge270) [reply docs/Internal Systems/Research & Findings/JekyllResearchTushar.md:3]: Same goes for the other files

**Review by ginge270 — CHANGES_REQUESTED** [review]
A general comment is that I would always try avoid doing PRs this big in future. Each seperate piece of work should be it's own PR. It makes it quiet challenging to review because it's so big - It's not as much of an issue with documentation but when it is code - you always want your changes to be as small as possible so if something goes wrong or breaks it is much easier to fix

File: docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+ 
+
+How will we structure our work During the Trimester? 
+
+We will run our tasks in a Sprint which will run over 2 Weeks, at the end of each sprint we will do a retrospective to see what we can have do differently for our next sprint. 
+
+ 
+
+
```
Comment (ginge270) [inline docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md:51]: Would fix up the spacing in this file

File: docs/Internal Systems/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Internal Systems Documentation
+
```
Comment (ginge270) [inline docs/Internal Systems/README.md:2]: I would be good to create a contents page here that hyperlinks to the research that has been uploaded

File: docs/Internal Systems/Research & Findings/Initial-Research.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
+# Initial Research for Internal Systems 
+_As we are beggining our work on Internal systems we need to generate some ideas for the areas that we can work on and the tools that we might use going forward in our project. This could include different coding language that we are farmiliar with and tools we have used. Different static site generators that exitst and how we can host out site once generated. From here we should also get an idea for the how we might manage Quality Assurance side of the product and mainting its quality._
+## Website Technologies
+
+#
```
Comment (talzid) [inline docs/Internal Systems/Research & Findings/Initial-Research.md:5]: Is there a reason for the blank headings throughout? 
If you want the dashed border, please use `---`

File: docs/Internal Systems/Research & Findings/StaticSiteGeneratorsAreeb.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+ 
+
+# 
+ <br>
+
+## Gatsby  
+<br>
+
+* Gatsby is a site generator based on JavaScript and React
```
Comment (talzid) [inline docs/Internal Systems/Research & Findings/StaticSiteGeneratorsAreeb.md:21]: Please use `-` as a bullent for consistency across all documentation

File: docs/Internal Systems/Research & Findings/StaticWebGeneratorsManveen.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+* Lastly, when compared with respect to documentation, and accessibility, Next.js competes well with Gatsby and Jekyll.  
+<br>
+## Cons: 
+<br>
+* However, the main distinction is that, while Next.js requires a server to work (in order to provide SSR capabilities), Gatsby and Jekyll can run without one. Also, Gatsby provides pure HTML/JS/CSS at build time, whereas Next.js generates HTML/JS/CSS at runtime. 
+
+<br>
+
+## [Infographic of comaprison](https://www.gatsbyjs.com/features/jamstack/gatsby-vs-nextjs-vs-jekyll)
```
Comment (talzid) [inline docs/Internal Systems/Research & Findings/StaticWebGeneratorsManveen.md:29]: Please link the url in a sentence, not a heading, and provide a sentence explaining the link.
This is an accessibility change.

File: docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
+## Internal Systems Retrospective Minutes
```
Comment (talzid) [inline docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md:1]: Should this not be a h1 heading `# ` 
same goes for the following files

**Review by talzid — CHANGES_REQUESTED** [review]
Thank you for the research. Glad to see it merged to Github. 
I have left a few suggestions for your review in addition to the changes request by Liz.

File: docs/Internal Systems/Research & Findings/JekyllResearchTushar.md (hunk 1 above)
Comment (Fletch99) [reply docs/Internal Systems/Research & Findings/JekyllResearchTushar.md]: This change has been made all files now no longer contain names in the title.

File: docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md (hunk 2 above)
Comment (Fletch99) [reply docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md]: Spacing has been fixed in this file

File: docs/Internal Systems/README.md (hunk 3 above)
Comment (Fletch99) [reply docs/Internal Systems/README.md:2]: I have initiated this but the link are not yet finalized they can be once git has been updated to the files I have created in this PR

File: docs/Internal Systems/Research & Findings/Initial-Research.md (hunk 4 above)
Comment (Fletch99) [reply docs/Internal Systems/Research & Findings/Initial-Research.md]: Thank you for this i have updated all files in the PR to use this instead.

File: docs/Internal Systems/Research & Findings/StaticSiteGeneratorsAreeb.md (hunk 5 above)
Comment (Fletch99) [reply docs/Internal Systems/Research & Findings/StaticSiteGeneratorsAreeb.md]: Done

File: docs/Internal Systems/Research & Findings/StaticWebGeneratorsManveen.md (hunk 6 above)
Comment (Fletch99) [reply docs/Internal Systems/Research & Findings/StaticWebGeneratorsManveen.md]: Fixed this is now no longer a heading.

File: docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md (hunk 7 above)
Comment (Fletch99) [reply docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md:1]: I made a change here but I may need more explanation on what is expected.

**Review by ginge270 — APPROVED** [review]

File: docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+<br>
```
Comment (tancnle) [inline docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md:2]: **issue:** Could we avoid using HTML tag in markdown (ie. `<br>` tag)? Line spacing is controlled by the markdown renderer.

File: docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+### Location:	Ms Teams
+### Date:		21/03/2022
+### Time:		6pm - 7:30pm
+### Minutes by:	Matthew Fletcher
```
Comment (tancnle) [inline docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md:7]: **suggestion:** These are, semantically, not headers. Could we convert them to a bullet list perhaps?

**Review by tancnle — CHANGES_REQUESTED** [review]
Thank you @Fletch99 for your work here ❤️. I have left some suggestions for your consideration. Please let me know what you think 🙏🏼.

Could you also please look at the failure in the build? _Hint_: it is something to do with extraneous number of empty lines at the end of the docs.

**Pull request closed without merging**